# add the EXTRA_FRAMEWORKS frameworks also added by the configure script

diff --git build/cmake/config.cmake build/cmake/config.cmake
index 90277f3955954cdfc9ef56012e26ee9e535b1fad..ad493297124a98ed8797f03f9c36d8b800836608 100644
--- build/cmake/config.cmake
+++ build/cmake/config.cmake
@@ -148,6 +148,9 @@ function(wx_write_config)
         set(WXCONFIG_CFLAGS "-pthread")
         set(WXCONFIG_LDFLAGS "-pthread")
     endif()
+    if(APPLE)
+        set(WXCONFIG_LDFLAGS "${WXCONFIG_LDFLAGS} -framework IOKit -framework Carbon -framework Cocoa -framework AudioToolbox -framework System -framework OpenGL")
+    endif()
     set(WXCONFIG_CPPFLAGS "-DWXUSINGDLL")
     foreach(flag IN LISTS wxTOOLKIT_DEFINITIONS)
         wx_string_append(WXCONFIG_CPPFLAGS " -D${flag}")
